deploy.sh

```diff
@@ -11,7 +11,8 @@ LOG_FILE=deployment.log
 #   PORT=8100 OPENFISCA_PORT=12100 ./deploy.sh aah
 # Example for production:
 #   PORT=8000 OPENFISCA_PORT=2000 PUBLIC_HOST=mes-aides.gouv.fr PROTOCOL=https ./deploy.sh
-TARGET_BRANCH=${1:-master}
+USER=`whoami`
+TARGET_BRANCH=${1:-${USER##mes-aides-}}
```

Comment:
The only time where I needed it was for a very long branch name, as there is a limit on the username size. So I had to use `mes-aides-irw` for `interrupted-ressources-wording, and actually use the extra argument.